site.yml

```diff
@@ -192,11 +192,21 @@
 - name: keystone setup
   hosts: controller[0]
   any_errors_fatal: true
+  pre_tasks:
```

Comment:
We should probably add the tags listed at the role to the be play level tags, since we've moved some of the tasks out of the role.